spot_rl_experiments/spot_rl/envs/test_cfslam.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+# black: ignore-errors
+import json
+import os
+
+
+def calculate_height(object_tag):
+    # Iterate through each dictionary in the world_graph
+    os.chdir(
+        "/home/tusharsangam/Desktop/spot-sim2real/spot_rl_experiments/spot_rl/envs"
+    )
+    with open("cfslam_object_relations.json") as f:
+        world_graph = json.load(f)
+    print("THE OBJECT I GOT IS", object_tag)
+    for rel in world_graph:
+        # Iterate through all keys in the dictionary
+        for key, value in rel.items():
+            if isinstance(value, dict) and value.get("object_tag") == object_tag:
```

Comment:
does this mean that the name of `object_tag` needs to be exact? what if you cannot fine the name? we just throw an error?